packages/sdk/tasks/one-time/arrakis-polygon-plugins.ts

```diff
@@ -128,7 +128,7 @@ task("arrakis:polygon:set-flywheels", "set plugin for each market")
       }
       try {
         //   Step 6: add Flywheels to market
-        await sdk.addFlywheelCoreToComptroller(mimoFlywheelAddress, COMPTROLLER, { from: signer.address });
+        await sdk.addFlywheelCoreToComptroller(mimoFlywheelAddress, COMPTROLLER);
```

Comment:
In line with these, this task must be updated as well  https://github.com/Midas-Protocol/monorepo/blob/development/packages/sdk/tasks/market/deploy-dynamic-rewards-market.ts